packages/perseus/src/widgets.ts

```diff
@@ -115,6 +118,8 @@ export const replaceDeprecatedEditors = () => {
     replaceEditor("sequence", "deprecated-standin");
     replaceEditor("simulator", "deprecated-standin");
     replaceEditor("unit-input", "deprecated-standin");
+
+    replaceEditor("input-number", "numeric-input");
 };
```

Comment:
Note to self: Add comment here explaining the editor page logic so that someone can follow the trail, if this is where they start of their journey